modules/my_health/app/controllers/my_health/v1/conditions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module MyHealth
+  module V1
+    class ConditionsController < MrController
+      def index
+        patient_id = params[:patient_id]
+        resource = client.list_conditions(patient_id)
+        raise Common::Exceptions::InternalServerError if resource.blank?
+
+        render json: resource.to_json
+      end
+
+      def show
+        condition_id = params[:id].try(:to_i)
+        resource = client.get_condition(condition_id)
+        raise Common::Exceptions::InternalServerError if resource.blank?
```

Comment:
I see you have some tests checking the successful response. Can you also test the error paths?

Reply:
I added a couple more tests for error paths. Let me know if you think anything else is needed. Thanks!